wg-serving/README.md

```diff
@@ -49,4 +49,42 @@ The WG Serving will operate in several workstreams with different areas of focus
 - [WS Autoscaling](https://docs.google.com/document/d/1QsN4ubjerEqo5L4bQamOFFS2lmCv5zNPis2Z8gcIITg/edit?usp=sharing)
 - WS MultiHost/MultiNode
 - [WS Orchestration](https://docs.google.com/document/d/1hbEx3ZEqdXCqWH9RL3uy9FIy35B8pFJ5KiK3HsOz2FE/edit?usp=sharing)
+
+
+## How to ...
+
+### subscribe to WG events
+
+1. Join the [`dev@kubernetes.io`](https://groups.google.com/a/kubernetes.io/g/dev) and [`wg-serving@kubernetes.io`](https://groups.google.com/a/kubernetes.io/g/wg-serving) for needed permissions.
+2. You should have meeting invites in your inbox after some delay. Make sure to fish them out of Junk/Spam and accept and invite so it will be added to the calendar
+3. Import [WG Serving Shared Calendar](https://calendar.google.com/calendar/u/0/embed?src=e896b769743f3877edfab2d4c6a14132b2aa53287021e9bbf113cab676da54ba@group.calendar.google.com) to see WG events.
+
+### schedule an ad-hoc or workstream meeting
+
+1. Get access to the [WG Serving Shared Calendar](https://calendar.google.com/calendar/u/0/embed?src=e896b769743f3877edfab2d4c6a14132b2aa53287021e9bbf113cab676da54ba@group.calendar.google.com). Ask chairs in #wg-serving slack chat.
+2. Create a meting nodes document or find existing one. Make sure it is shared as following:
+   - Shared with `dev@kubernetes.io` for commenting
+   - Shared with `wg-serving@kubernetes.io` for editing
+3. Create an event. Make sure to create it in the shared calendar.
+   - Use this zoom link: https://zoom.us/j/9955436256?pwd=Z2FQWU1jeDZkVC9RRTN4TlZyZTBHZz09
+   - Invite `wg-serving@kubernetes.io`
+   - Invite `calendar@kubernetes.io` so the event will appear in https://www.kubernetes.dev/resources/calendar/
+   - Mention the meeting notes document in the description
+4. Make sure one of the chairs has accepted the invite. If not - ask chairs for a host key to claim a host during the meeting.
+5. Start a recording of the meeting (record to the cloud).
+
```

Comment:
Missing an end recording step? (not sure if it's required since end the meeting might automatically end the recording)

Reply:
it will end when meeting ends

Reply:
```suggestion
6. You can end the recording explicitly or it will be finished when the meeting will end and all participants will leave it
```